app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt

```diff
@@ -130,6 +131,8 @@ class Message : RealmObject {
     var isDeletedOnApi: Boolean = false
     @Transient
     var latestCalendarEventResponse: CalendarEventResponse? = null
+    @Transient
+    var swissTransferFiles: RealmList<SwissTransferFile> = realmListOf()
```

Comment:
We could had a field ignored and transient, along the line of `val hasSwissTransferFiles get() = swissTransferFiles.isNotEmpty()` or `val hasSwissTransferFiles get() = swissTransferUuid != null`